### PR TITLE
Added queue_using_stack in Javascript and cpp

### PR DESCRIPTION
**Pull request by khushal87**
Refer issue #658

### CODE REVIEW

File: Queue_Using_Stacks/Queues_Using_Stacks.cpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+	cout << q.deQueue() << '\n'; 
+	cout << q.deQueue() << '\n'; 
+
+	return 0; 
+} 
+
+// OUTPUT => 
+// 1
+// 2
```
Comment (jainaman224) [inline Queue_Using_Stacks/Queues_Using_Stacks.cpp]: Add new line at the end of file.

File: Queue_Using_Stacks/Queues_Using_Stacks.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+class CrazyQueue {
```
Comment (jainaman224) [inline Queue_Using_Stacks/Queues_Using_Stacks.js]: Why name of the class is CrazyQueue?

File: Queue_Using_Stacks/Queues_Using_Stacks.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+myQueue.enqueue(4);
+myQueue.enqueue(5);
+myQueue.peek();
+myQueue.dequeue();
+myQueue.dequeue();
+myQueue.dequeue();
+
+
+// OUTPUT:-
+// => CrazyQueue { first: [ 5, 4 ], last: [] }
```
Comment (jainaman224) [inline Queue_Using_Stacks/Queues_Using_Stacks.js]: Add new line at the end of file.

**Comment by khushal87** [comment]
Done @jainaman224

**Comment by jainaman224** [comment]
@khushal87 Please learn about [commit squashing](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit)

**Pull request merged**